fpr/tests.py

```diff
@@ -0,0 +1,113 @@
+import uuid
+
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+from fpr.forms import FPRuleForm, IDToolForm
+from fpr.models import (
+    Format, FormatGroup, FormatVersion,
+    FPCommand, FPRule, FPTool,
+    IDCommand, IDTool
+)
+from fpr.utils import get_revision_descendants
+
+
+class TestUtils(TestCase):
+
+    def test_get_revision_descendants(self):
+        cmd1 = IDCommand.objects.create(
+            uuid='37f3bd7c-bb24-4899-b7c4-785ff1c764ac')
+        cmd2 = IDCommand(description='Foobar')
+        cmd2.save(replacing=cmd1)
+
+        descendants = get_revision_descendants(IDCommand, cmd1.uuid, [123])
+        self.assertEqual(descendants, [123, cmd2])
+
+        with self.assertRaises(IDCommand.DoesNotExist):
+            descendants = get_revision_descendants(
+                IDCommand, '12345', [])
```

Comment:
Can you provide more detail on how to run these tests? I was unable to do so easily.

Also, looking at the definition of `get_revision_descendants` it looks like that func will not raise the exception that you are expecting to catch here. It looks like it catches that exception and just passes. 

Relatedly, I noticed that travis is not running these tests. It only runs those in testproject/. If this were fixed, then we could see if these tests were passing.